parm/parm_gdas/land_jedi_fix_gfsv17.yaml

```diff
@@ -0,0 +1,7 @@
+mkdir:
+- $(DATA)/fv3jedi
+copy:
+- [$(HOMEgfs)/fix/gdas/fv3jedi/fv3files/akbk$(npz).nc4, $(DATA)/fv3jedi/akbk.nc4]
+- [$(HOMEgfs)/fix/gdas/fv3jedi/fv3files/fmsmpp.nml, $(DATA)/fv3jedi/fmsmpp.nml]
+- [$(HOMEgfs)/fix/gdas/fv3jedi/fv3files/field_table_gfdl, $(DATA)/fv3jedi/field_table]
+- [$(HOMEgfs)/ush/gfs-land_gfsv17.yaml, $(DATA)/fv3jedi/gfs-land.yaml]
```

Comment:
Why is this in `ush`?  This should be a file in `$(HOMEgfs)/parm/parm_gdas/...`

Reply:
I think we can handle this case in the `land_jedi_fix.yaml`  itself instead of adding this yaml.